OpenSim/Simulation/Model/OffsetFrame.h

```diff
@@ -242,14 +248,28 @@ void OffsetFrame<C>::constructConnectors()
 }
 
 //=============================================================================
-// FRAME COMPUTATIONS
+// FRAME COMPUTATIONS must be specialized by concrete derived Offsets
 //=============================================================================
 // Implementation of Frame interface by OffsetFrame.
 template <class C>
 SimTK::Transform OffsetFrame<C>::
-calcGroundTransform(const SimTK::State& s) const
+calcTransformInGround(const SimTK::State& state) const
 {
-    return getParentFrame().getGroundTransform(s)*getOffsetTransform();
+    return SimTK::Transform(SimTK::Vec3(SimTK::NaN));
+}
+
+template <class C>
+SimTK::SpatialVec OffsetFrame<C>::
+calcVelocityInGround(const SimTK::State& state) const
+{
+    return SimTK::SpatialVec(SimTK::Vec3(SimTK::NaN));
+}
+
+template <class C>
+SimTK::SpatialVec OffsetFrame<C>::
+calcAccelerationInGround(const SimTK::State& state) const
+{
+    return SimTK::SpatialVec(SimTK::Vec3(SimTK::NaN));
```

Comment:
Why do the two methods above need to be implemented in `OffsetFrame`? Could OffsetFrame just not implement them?